init.rb

```diff
@@ -48,6 +48,18 @@
 
 current_flavor = Rock.flavors.current_flavor
 
+#The following option is used to determine which xml importer should be used.
+#Currently castxml is working
+
+configuration_option('TYPELIB_CXX_LOADER', 'string',
+    :default => 'castxml',
```

Comment:
Keyword argument syntax is `key: value`. `:key => value` only works as a backward compatibility feature.